README.rst

```diff
@@ -56,14 +57,14 @@ Usage
   
   {% post_to_linkedin <subject> <object_or_url> <link_text> %}
   
-  {% post_to_mail <email_subject> <text_to_post> <object_or_url> <link_text> %}
-  
```

Comment:
why was this line deleted?

Reply:
I see, `post_to_mail` was renamed to `send_to_mail`, and the `post_to_mail` was not removed from the readme. Nevermind :)